packages/react-error-overlay/package.json

```diff
@@ -40,16 +41,17 @@
     "babel-eslint": "10.1.0",
     "babel-jest": "^24.9.0",
     "babel-loader": "8.1.0",
-    "babel-preset-react-app": "^9.1.2",
+    "babel-preset-react-app": "^9.1.3",
```

Comment:
Oh wait - this is because of the version bumps in other packages 🤦 ... ignore me...

I guess it adds to the confusion which is why it'd be great if we didn't bump those versions...